File: Introductory Problems/Repetitions/repetitions.java
import java.util.Scanner;

public class repetitions{
    private static void solve(){
        Scanner scanner = new Scanner(System.in);
        String s = scanner.nextLine();
        int n = s.length(), l = 0, r = 0, temp = 0;
        while(r<n){
            while(r<n && s.charAt(l) == s.charAt(r))
                r++;
            temp = Math.max(temp, r-l);
            l = r;
        }
        System.out.println(Math.max(temp, r-l));
    }

    public static void main(String args[]){
        Scanner scanner = new Scanner(System.in);
        long testCases = 1;
        // long testCases = scanner.nextLong();
        while (testCases-- > 0) {
            solve();
        }
        scanner.close();
    }
}

File: Introductory Problems/Repetitions/repetitions.cpp
#include<bits/stdc++.h>
using namespace std;
#define ll long long int

void solve(){
	string s = "abc";
	cin>>s;
	ll n = s.size(), l = 0, r = 0, temp = 0;
	if(n == 1){
		cout<<"1";
		return;
	}
	while(r<n){
		while(r<n && s[l] == s[r]){
			r++;
		}
		temp = max(temp, r-l);
		l=r;
	}
	cout<<max(temp, r-l);
	return;
}

int main(){
	long long testCases;
	testCases = 1;
	// cin>>testCases;
	while(testCases--){
		solve();
	}
	return 0;
}